streams/src/test/java/org/apache/kafka/streams/integration/SmokeTestDriverIntegrationTest.java

```diff
@@ -104,10 +104,6 @@ public void shouldWorkWithRebalance() throws InterruptedException {
             clients.add(smokeTestClient);
             smokeTestClient.start(props);
 
-            while (!clients.get(clients.size() - 1).started()) {
-                Thread.sleep(100);
-            }
```

Comment:
This isn't needed anymore, as `SmokeTestClient#start` now blocks until the instance goes to running the first time.